contracts/MassMigs.sol

```diff
@@ -0,0 +1,170 @@
+pragma solidity ^0.5.15;
+pragma experimental ABIEncoderV2;
+
+import { FraudProofHelpers } from "./FraudProof.sol";
+import { Types } from "./libs/Types.sol";
+import { RollupUtils } from "./libs/RollupUtils.sol";
+import { Tx } from "./libs/Tx.sol";
+import { MerkleTreeUtilsLib } from "./MerkleTreeUtils.sol";
+import "@nomiclabs/buidler/console.sol";
+
+contract MassMigs is FraudProofHelpers {
```

Comment:
Can we rename all "MassMigs" to "MassMigration"? Want the variables to be easy to pronounce so we can communicate better.

These places need the rename:

- MassMigs.sol
- contract MassMigs
- MassMig struct in Tx.sol